start_env.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+export DATABASE_URL=postgresql://postgres:somePassword@localhost/postgres
+export SECRET_KEY=someSecretStringUsedByFlask
+export APP_SETTINGS=config.DevelopmentConfig
+export FLASK_ENV=development
+export GOOGLE_APPLICATION_CREDENTIALS=secretKey.json
```

Comment:
How about we put here GOOGLE_APPLICATION_CREDENTIALS=<path to secret key> and in DATABASE URL postgresql://postgres:<your local db password>@localhost/postgres ? I just don't feel comfortable that it's the same for everyone and with this default values while it's an open source, because it might find its way to prod and then it's just an exposed password.